### ISSUE TITLE
Is this project still being maintained?

### ISSUE THREAD

**Issue by bkuhl** [Xethron/migrations-generator#178]
Seems like this project has quite a few PRs and issues without any maintainer commenting on them.  Is this still being maintained?

**Comment by scoutman57** [comment]
I am pretty sure it's not. Nothing since Sept 2017

**Comment by geoffreyvanwyk** [comment]
Perhaps somebody else should take over or fork?

**Comment by nathanaelytj** [comment]
Anybody taking over this project?

**Comment by eugenefvdm** [comment]
This is a very relevant question, but there is a bigger question, and that is:

1. Does the software still work as intended?
2. Has much changed? Maybe it doesn't need too much maintaining as it just works.

But at the end of the day, it would be nice to know what the status is because this is extremely valuable software.

**Comment by bkuhl** [comment]
You're right.  I look at the number of issues, a decent number of which appear to be pretty simple issues and go without any kind of response from a contributor.  I avoided using this project for that reason.

**Comment by scoutman57** [comment]
everyone,

This project is not maintained anymore. It was abandoned long ago.

**Comment by willsmanley** [comment]
For those who have abandoned, what is the new preferred package for creating migrations from DB tables?

**Comment by bkuhl** [comment]
Beginning with Laravel 8, this kind of package isn't needed: https://twitter.com/taylorotwell/status/1247561693946839040
